BLSQ/iaso
# IA-2992: View only permission for Planning

Explain what problem this PR is resolving
- It would be useful to allow some users to visualize plannings without having any right to modify those plannings.
Related JIRA tickets : [IA-2992](https://bluesquare.atlassian.net/browse/IA-2992)

## Self proofreading checklist


## Changes
- Add iaso_planning_read and iaso_planning_write permissions
- create migration to link user with iaso_registry and iaso_planning with the new permissions
- Display Edit, delete, copy, create when the user has iaso_planning_write

## How to test
- Edit or create a user and remove the iaso_planning_write permission
- check if the edit, delete, duplicate and create buttons are there anymore
## Print screen / video
[Screencast from 2024-05-30 18-12-43.webm](https://github.com/BLSQ/iaso/assets/19631540/d55899b3-6cdd-41f7-96a0-ee6929442c18)


[IA-2992]: https://bluesquare.atlassian.net/browse/IA-2992?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
